client/doc/onboarding.md

```diff
@@ -0,0 +1,64 @@
+## Onboarding component
+
+### Onboarding class
+
+This class, located in `lib/onboarding`, is an agnostic and framework free onboarding manager. It manages the different steps of onboarding and handle event triggered by steps object.
+
+The goal of this class is facilitate migration to another framework in the future.
+
+At this time, the framework used is Backbone/Marionette and Onboarding is implemented like a classical POO object.
+
+## Step class
```

Comment:
I dont understand nature of this class; is this the state of the application? is this a model?